evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -243,6 +264,25 @@ def test_fit_predict_sk_failure(
     assert y_pred.index.equals(X_test.index)
 
 
+def test_arima_sp_changes_result():
+    y = pd.Series([math.sin(i) for i in range(200)])
+
+    X = pd.DataFrame({"dates": pd.date_range("2021-01-01", periods=200, freq="D")})
```

Comment:
can we check that `sp` is set correctly for the two examples here? I.e `sp == 7` and `sp==4`?